bundle/language_en.properties

```diff
@@ -321,6 +323,10 @@ angal.billbrowser.today.btn
 angal.billbrowser.today.btn.key                                                  = T
 angal.groupsbrowser.groupmenu.btn                                                = GroupMenu
 angal.groupsbrowser.groupmenu.btn.key                                            = M
+angal.lab.n.btn                                                                  = N
+angal.lab.p.btn                                                                  = P
```

Comment:
ok, these are "negative" and "positive" abbeviations

Reply:
Done